pyseir/inference/model_plotting.py

```diff
@@ -0,0 +1,263 @@
+from datetime import timedelta
+from copy import deepcopy
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from pyseir.load_data import HospitalizationDataType  # Do we still need this?
+from pyseir.utils import get_run_artifact_path, RunArtifact
+
+
+def plot_fitting_results(result) -> None:
+    """
+
+    """
+    # Entry point from model_fitter
+    output_file = get_run_artifact_path(result.fips, RunArtifact.MLE_FIT_REPORT)
+
+    # Save the mle fitter
+    fig = result.mle_model.plot_results()
+    fig.savefig(output_file.replace("mle_fit_results", "mle_fit_model"), bbox_inches="tight")
+
+    # Save the model in log mode
+    fig = _plot_ModelFitter_results(result)
+    fig.gca().set_yscale("log")
+    fig.savefig(output_file, bbox_inches="tight")
+
+    # Save the model in linear mode
+    fig = _plot_ModelFitter_results(result)
+    fig.gca().set_yscale("linear")
+
+    fig.savefig(
+        output_file.replace("mle_fit_results", "mle_fit_results_linear"), bbox_inches="tight"
+    )
+    return
+
+
+def _plot_ModelFitter_results(result) -> plt.Figure:
```

Comment:
I'd prefer to name this `_plot_model_fitter_results` to keep inline with standard python names